tests/test_rooms/test_room_analytics.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+from tests.base import BaseTestCase
+from fixtures.token.token_fixture import (admin_api_token)
+from fixtures.room.room_analytics_fixtures import (
+    get_most_used_room_in_a_month_analytics_query,
+    get_most_used_room_in_a_month_analytics_response,
+    most_used_room_in_a_month_analytics_invalid_location_query,
+    most_used_room_in_a_month_analytics_invalid_location_response,
+    get_least_used_room_per_week_query,
+    get_least_used_room_per_week_response,
+    get_least_used_room_without_event_query,
+    get_least_used_room_without_event_response
+
+)
+
+
+class QueryRoomsAnalytics(BaseTestCase):
+
+    api_headers = {"Authorization": "Bearer" + " " + admin_api_token}
```

Comment:
Since we used the name headers in the other sections of the code and you have decided to use the same format, it would be great to just use the same variable name, not to reference same varaiable but fr uniformity